src/components/Kv/KvCarousel.vue

```diff
@@ -1,24 +1,161 @@
 <template>
-	<div class="kv-carousel">
-		<slot></slot>
+	<div
+		class="kv-carousel"
+		:class="carouselModifierClasses"
+		ref="KvCarousel"
+	>
+		<div class="kv-carousel__slides"
+			@mouseover="pauseOnHover ? paused = true : paused = false"
+			@touchstart="pauseOnHover ? paused = true : paused = false"
+			@mouseleave="pauseOnHover ? paused = false : paused = false"
+			@touchend="pauseOnHover ? paused = false : paused = false"
```

Comment:
For `@mouseleave` and `@touchend` you don't have to check `pauseOnHover` because you're setting `paused` to false either way.